src/Rules/StarterWeb/AI/IndividualAuth/appsettings.json

```diff
@@ -2,11 +2,7 @@
   "ApplicationInsights": {
     "InstrumentationKey": ""
   },
-  "Data": {
-    "DefaultConnection": {
-      "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true"
-    }
-  },
+  "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true",
```

Comment:
Can we keep the data section? It was something we had agreed upon earlier when reviewing templates. How  about something like the following?
```C#
"Data" :{ "WebAppNameConnectionString":"<value>"}
or
"Data" :{ "ConnectionString":"<value>"}
```